<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/p1197456005.webp.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_addMusic">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border:1px solid lightgray; background:rgb(230,230,230);}
QPushButton:hover{border-color:blue; background:rgb(27, 96, 149);}
</string>
    </property>
    <property name="text">
     <string>ADD MUSIC</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_volume">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>70</y>
      <width>31</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
    border-color: #cbcbcb;
}
QSlider::groove:vertical {  

background: #cbcbcb;  
width: 6px;  
border-radius: 1px;  
padding-left:-1px;  
padding-right:-1px;  
padding-top:-1px;  
padding-bottom:-1px;  
} 
QSlider::sub-page:vertical {  
background: #cbcbcb;  

border-radius: 2px;  
} 
QSlider::add-page:vertical {  
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,   
    stop:0 #439cf4, stop:1 #439cf4);  
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,  
    stop: 0 #439cf4, stop: 1 #439cf4);  
width: 10px;  
border-radius: 2px;  
}
QSlider::handle:vertical   
{     
    
	border-image: url(:/img/dot.png);
     margin: -2px -7px -2px -7px; 
    height: 17px;
}  </string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="TextTicker" name="label_musictitle">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="slider_music">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
border: 1px solid #4A708B;
background: #C0C0C0;
height: 5px;
border-radius: 1px;
padding-left:-1px;
padding-right:-1px;
}
 
QSlider::sub-page:horizontal {
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
    stop:0 #B1B1B1, stop:1 #c4c4c4);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #5DCCFF, stop: 1 #1874CD);
border: 1px solid #4A708B;
height: 10px;
border-radius: 2px;
}
 
QSlider::add-page:horizontal {
background: #575757;
border: 0px solid #777;
height: 10px;
border-radius: 2px;
}
 
QSlider::handle:horizontal 
{
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, 
    stop:0.6 #45ADED, stop:0.778409 rgba(255, 255, 255, 255));
 
    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}
 
QSlider::handle:horizontal:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.6 #2A8BDA, 
    stop:0.778409 rgba(255, 255, 255, 255));
 
    width: 11px;
    margin-top: -3px;
    margin-bottom: -3px;
    border-radius: 5px;
}
 
QSlider::sub-page:horizontal:disabled {
background: #00009C;
border-color: #999;
}
 
QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}
 
QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>420</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00 / 00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-image: url(:/img/close_1.png);
background-color:transparent;
}
QPushButton:hover{
background-image: url(:/img/close_2.png);
background-color:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>550</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-image: url(:/img/stop.png);
background-color:transparent;}
QPushButton:hover{
	background-image: url(:/img/stop_ho.png);
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_play">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-image: url(:/img/play.png);
background-color:transparent;}
QPushButton:hover{background-image: url(:/img/play_ho.png);
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_next">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-image: url(:/img/next.png);
background-color:transparent;}
QPushButton:hover{
	background-image: url(:/img/next_ho.png);
background-color:transparent;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_last">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-image: url(:/img/last.png);
background-color:transparent;}
QPushButton:hover{
	background-image: url(:/img/last_ho.png);
background-color:transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_valume">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/valume.png);
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_playBackMode" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QPushButton" name="button_loop">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/img/loop.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_currentItemInLoop">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/img/currentItemInLoop.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_sequential">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/img/sequenti.png);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_random">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
background-image: url(:/img/random.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_playBackMode">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>530</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
background-image: url(:/img/loop.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-image: url(:/img/title.png);
}
QLabel:hover
{
background-image: url(:/img/title_ho.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/title_string_2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{border:1px solid gray; background-color:transparent; }
QListWidget::Item{padding-top:20px; padding-bottom:4px; }
QListWidget::Item:hover{background:skyblue; }
QListWidget::item:selected{background:lightgray; color:white; }
QListWidget::item:selected:!active{border-width:0px; background:lightgreen; }</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_hide">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-image: url(:/img/hide.png);
background-color:transparent;
}
QPushButton:hover{
background-image: url(:/img/hide_ho.png);
background-color:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>button_addMusic</zorder>
   <zorder>slider_volume</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>slider_music</zorder>
   <zorder>label_time</zorder>
   <zorder>button_close</zorder>
   <zorder>button_stop</zorder>
   <zorder>button_play</zorder>
   <zorder>button_next</zorder>
   <zorder>button_last</zorder>
   <zorder>button_valume</zorder>
   <zorder>widget_playBackMode</zorder>
   <zorder>button_playBackMode</zorder>
   <zorder>label</zorder>
   <zorder>listWidget</zorder>
   <zorder>label_title</zorder>
   <zorder>button_hide</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TextTicker</class>
   <extends>QLabel</extends>
   <header>textticker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
